internal/exec/engine.go

```diff
@@ -75,6 +81,23 @@ func (e Engine) Run(stageName string) error {
 		return err
 	}
 
+	baseConfigFound := true
+	if err == providers.ErrNoProvider {
+		baseConfigFound = false
+	}
+
+	configExists, err := executil.PathExists(e.ConfigCache)
+	if err != nil {
+		e.Logger.Info("failed to read cached config: %v", err)
```

Comment:
Hmm, I think the message here should be something like `failed to check for %s: %v` ?